### PR TITLE
feat(dataStore_pref): show all the pref data and a sample

### LINKED ISSUE #53: explore DataStore Preferences
**Is your feature request related to a problem? Please describe.**
- Support DataStore preferences along the SharedPreferences

discussion reference : https://github.com/mocklets/pluto/discussions/10

### PR DESCRIPTION
**Pull request by rishabhsinghbisht**
- [x] show data from data store preferences.
- [x] allow to edit data.
- [x] add noop
- [ ] add type checks if able to detect types.
- [ ] allow to add/remove key value pair.

### CODE REVIEW

File: datastore-pref-plugin-no-op/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.pluto.plugin.datastore.pref">
```
Comment (srtvprateek) [inline datastore-pref-plugin-no-op/src/main/AndroidManifest.xml]: make it `com.pluto.plugins.datastore.pref`

File: datastore-pref-plugin-no-op/src/main/java/com/pluto/plugin/datastore/pref/DataStorePrefPlugin.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import androidx.fragment.app.Fragment
+import com.pluto.plugin.Plugin
+import com.pluto.plugin.PluginConfiguration
+import java.lang.IllegalStateException
+
+class DataStorePrefPlugin : Plugin("Bisht") {
+
+    override fun getConfig() = PluginConfiguration(
+        name = "DataStore - preference",
+        version = "0.1-beta"
```
Comment (srtvprateek) [inline datastore-pref-plugin-no-op/src/main/java/com/pluto/plugin/datastore/pref/DataStorePrefPlugin.kt]: can take this from `BuildConfig`

**Review by srtvprateek — CHANGES_REQUESTED** [review]
.

File: datastore-pref-plugin-no-op/src/main/java/com/pluto/plugin/datastore/pref/DataStorePrefPlugin.kt (hunk 2 above)
Comment (rishabhsinghbisht) [reply datastore-pref-plugin-no-op/src/main/java/com/pluto/plugin/datastore/pref/DataStorePrefPlugin.kt]: @srtvprateek is it build specific?

**Review by srtvprateek — APPROVED** [review]

**Pull request merged**